### PR TITLE
Fix amc example

### PR DESCRIPTION
**Pull request by chicm-ms**

### CODE REVIEW

**Comment by QuanluZhang** [comment]
@chicm-ms could you add amc pruner in the pruners table in our doc?

File: examples/model_compress/amc/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+```
+Once finished, saved checkpoint file can be found at:
+```
+logs/mobilenetv2_cifar10_train-run1/ckpt.best.pth.tar
+```
+
+## Pruning with AMCPruner
+Run following command to prune the trained model:
+```bash
+python amc_search.py --model_type mobilenetv2 --ckpt logs/mobilenetv2_cifar10_train-run1/ckpt.best.pth.tar
```
Comment (QuanluZhang) [inline examples/model_compress/amc/README.md:17]: python -> python3

File: examples/model_compress/amc/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+```
+Once finished, pruned model and mask can be found at:
+```
+logs/mobilenetv2_cifar10_r0.5_search-run2
+```
+
+## Finetune pruned model
+Run `amc_train.py` again with `--ckpt` and `--mask` to finetune the pruned model:
+```
+python amc_train.py --model_type mobilenetv2 --ckpt logs/mobilenetv2_cifar10_r0.5_search-run2/best_model.pth --mask logs/mobilenetv2_cifar10_r0.5_search-run2/best_mask.pth --n_epoch 100
```
Comment (QuanluZhang) [inline examples/model_compress/amc/README.md:27]: python -> python3

File: examples/model_compress/amc/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+```
```
Comment (QuanluZhang) [inline examples/model_compress/amc/README.md:28]: might be better to add model speedup as the last step

File: examples/model_compress/amc/README.md (hunk 1 above)
Comment (chicm-ms) [reply examples/model_compress/amc/README.md]: Thanks, fixed.

File: examples/model_compress/amc/README.md (hunk 2 above)
Comment (chicm-ms) [reply examples/model_compress/amc/README.md]: Thanks, updated.

File: examples/model_compress/amc/README.md (hunk 3 above)
Comment (chicm-ms) [reply examples/model_compress/amc/README.md:28]: Thanks, README updated to say speedup is done in last step.

**Comment by chicm-ms** [comment]
> @chicm-ms could you add amc pruner in the pruners table in our doc?

Thanks, fixed.

**Review by QuanluZhang — APPROVED** [review]

**Review by zheng-ningxin — APPROVED** [review]
Looks great~

**Review by colorjam — APPROVED** [review]

**Pull request merged**